### DISCUSSION TITLE
KSGER C245 C210 compatibility

### DISCUSSION THREAD

**Discussion by Stephan-4711** [deividAlfa/stm32_soldering_iron_controller#136] [Q&A]
Hi,

I bought a KSGER C245 C210 soldering station, on the first look the Mainboard could be compatible maybe. But even if is the firmware compatible with the Soldering iron then? Sorry I wasn't able to figure out this part

**Comment by deividAlfa** [comment]
Sorry, I don't fully understand the question.
Are you asking if the board is compatible?
No way to know with only a message, attach some pictures of it.

Also, I don't think there's any firmware backup for this model, so rolling back to original will be impossible, consider extracting it using pico-pwner attack as describes in the readme.

Cheers!

**Reply by Stephan-4711** [reply]
Now I disassembled the board. It looks different a lot 🙈 yea I wanted to know if it's compatible, the usage is a little Getting 
![PXL_20241009_045455304](https://github.com/user-attachments/assets/420503c2-19ad-4d7a-8e7a-cb1def14fbfb)
![PXL_20241009_045440629](https://github.com/user-attachments/assets/c9235887-8904-4b76-b5c5-1ed30488bd15)
![PXL_20241009_045431024](https://github.com/user-attachments/assets/d8ae4ea7-e354-47cf-87e8-47580f9e4b2c)

**Reply by deividAlfa** [reply]
It's very different, Nuvoton MCU, LCD screen... Won't work, sorry!

Can you make a picture of the mcu?
